JavaScript/JavaScriptSDK/SendBuffer.ts

```diff
@@ -132,6 +136,17 @@ module Microsoft.ApplicationInsights {
         }
 
         public enqueue(payload: string) {
+            if (this._buffer.length >= SessionStorageSendBuffer.MAX_BUFFER_SIZE) {
+                // sent internal log only once per page view
+                if (!this._bufferFullMessageSent) {
+                    _InternalLogging.throwInternalUserActionable(LoggingSeverity.WARNING,
+                        new _InternalLogMessage(_InternalMessageId.USRACT_SessionStorageBufferFull,
+                            "Maximum buffer size reached: " + this._buffer.length));
+                    this._bufferFullMessageSent = true;
```

Comment:
When/where does it get set back to false?
I presume that once the buffer has space again that this should no longer be true.